.github/workflows/build-with-bal-test-graalvm.yml

```diff
@@ -35,3 +35,5 @@ jobs:
       lang_tag: ${{ inputs.lang_tag }}
       lang_version: ${{ inputs.lang_version }}
       native_image_options: '-J-Xmx7G ${{ inputs.native_image_options }}'
+      # TODO : Enable after fixing this issue : https://github.com/ballerina-platform/ballerina-lang/issues/38882
+      additional_windows_build_flags: '-x test'
```

Comment:
We only need to disable the `locale` test right? not all of the tests

Reply:
This is a separate concern. It is a known issue of windos and graalvm

https://github.com/ballerina-platform/ballerina-lang/issues/38882